CAAppKitBridge/Demo/main.m

```diff
@@ -34,15 +34,15 @@ @interface DemoController : NSObject
 @end
 
 @implementation DemoController
-- (void) applicationDidFinishLaunching: (id) t
+- (void) applicationDidFinishLaunching: (id)t
```

Comment:
You have not moved `DemoController` interface nor implementation to its own file.

Reply:
Changed my mind, moved it to its own file.